catalog/to-consul/resource_test.go

```diff
@@ -2268,16 +1258,16 @@ func TestServiceResource_clusterIPTargetPortNamed(t *testing.T) {
 	actual := syncer.Registrations
 	require.Len(actual, 2)
 	require.Equal("foo", actual[0].Service.Service)
-	require.Equal("1.2.3.4", actual[0].Service.Address)
+	require.Equal("1.1.1.1", actual[0].Service.Address)
 	require.Equal(2000, actual[0].Service.Port)
 	require.Equal("foo", actual[1].Service.Service)
-	require.Equal("2.3.4.5", actual[1].Service.Address)
+	require.Equal("2.2.2.2", actual[1].Service.Address)
 	require.Equal(2000, actual[1].Service.Port)
 	require.NotEqual(actual[0].Service.ID, actual[1].Service.ID)
 }
 
-// testService returns a service that will result in a registration.
-func testService(name string) *apiv1.Service {
+// testLBService returns a service that will result in a registration.
```

Comment:
```suggestion
// testLBService returns a Kubernetes service of type LoadBalancer.
```

Reply:
Should we remove the "test" prefix? Doesn't make sense in this context at least to me. Maybe `lbService` or `lbServiceFixture`

Reply:
Yes, I thought so too. Will change!